frontend/components/base/BaseButton.vue

```diff
@@ -134,6 +134,11 @@ export default {
   line-height: 13px;
   padding: 9px 18px;
 }
+.tiny {
```

Comment:
rename "tiny" by something more specific. It will replace the class "small" ?

Reply:
For the moment I will put the styles only in the similarity button and in the future maybe assign a class for the same type